app/auth/signup/email/schema.ts

```diff
@@ -0,0 +1,15 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import * as z from 'zod'
+
+const signUpSchema = z
+  .object({
+    email: z.string().email(),
+    password: z.string().min(8),
+    confirmationPassword: z.string().min(8)
```

Comment:
The supabase password required number, capital letter or symbol too? If so,  you should add the validation here too

Reply:
There's no password strength restrictions by default, so we can just decide whatever we want.
So I'd say we have to decide password policy sooner or later by publishing this app to public.